src/main/scala/com/databricks/sparkdl/ImageUtils.scala

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2017 Databricks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.sparkdl
+
+import java.awt.image.BufferedImage
+import java.awt.{Color, Image}
+
+import org.apache.spark.image.ImageSchema
+import org.apache.spark.sql.Row
+
+object ImageUtils {
```

Comment:
@smurching will this show up in docs because it's not `private[sparkdl]` ?

Reply:
Probably

Reply:
It'll show up by default, although we can hide `ImageUtils` from the scaladocs if it needs to be exposed so that the Python API can call its methods.

Reply:
I can make it private for now, we can change it later or expose parts of it through a "python interface" type object.